### PR TITLE
translations: part 1

### PR DESCRIPTION
**Pull request by CvH**
Part one of changes to allow proper translations of our addons
- converting all `string.xml` to `string.po` and change all existing po files to an standard layout (I hope that layout is okay)
- all languages variables used at addons are starting now with `id=30000` (Kodi is doing partly the same)
- there should be no visible difference at the addons (if so I missed something while changing)

gh diff is a bit messy

### CODE REVIEW

**Comment by vpeter4** [comment]
Why removing german strings? Are they going to be put on transifex? Or they will have to be translated again by someone?

**Comment by CvH** [comment]
@vpeter4 they need to be added again manually to whatever translating frontend we are choosing (likely no transifex as it is lacking of basic functions like branches support, user rights management, build in git support ...) as the translated strings are "outsourced" to an LE/translation.git repo (or we would mess this repo with translation commits). To add them again it is just an copy/paste job.

**Comment by CvH** [comment]
updated, would be nice if this could go in before next mass addon rebuild (if there are somewhere problems better get it asap)

**Comment by CvH** [comment]
rebased

**Comment by CvH** [comment]
to conclude the translation mess :) @davu @chewitt @lrusak @HiassofT @sky42src 

to make this work at weblate (every following commit is heavily outdated)
- we need to convert every `strings.xml` to `strings.po`, basically done today (~8 still missing, already done at my repo somewhere)
- create an repository that holds every translation, move all translation files to that repo, demo repo https://github.com/CvH/translation (could be at the main repo too, doesn't matter too much, extra repo would be easier)
- a translation package that is pulled at addon creation https://github.com/CvH/LibreELEC.tv/commit/5ff192c3467faa4ed6dd29deeb052f4b0e15c84f

then the translation package is pulled if you build an addon and fills in every available translation without manual interaction

how would it work
- every addon needs a new "project" at weblate with basic setup (screenshot of translations, name and path of the main language file of the addon)
- otherwise this maintains itself, language addon changes (just the english master file) need to be pred to the translation repo OR setup some cron/whatever that pulls the current language file from the LE repo
- translations that are done at weblate (after confirming the changes) are pushed to the translations repo for example https://github.com/CvH/translation/commit/3977c09bd03a70299bb241aef090511ae8db1119 or https://github.com/CvH/translation/commit/9d97b5cdee6304c84fd99963a10c769dd9498e9f
- branches are supported
- automatic pr creation to the LE mai nrepo could be done too (iirc weblate supports that too)

File: packages/addons/driver/hdhomerun/source/resources/language/English/strings.po [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Kodi Media Center language file
```
Comment (enen92) [inline packages/addons/driver/hdhomerun/source/resources/language/English/strings.po:1]: It's be nice to also have the language layout fixed (e.g. ../resources/language/resource.language.en_gb/strings.po)

There's an add-on you can use to automate it all: https://gitlab.com/ronie/script.language-convert

Comment (CvH) [reply packages/addons/driver/hdhomerun/source/resources/language/English/strings.po:1]: tx for hint, currently it is still an 3yr old pr that is heavily outdated and needs rework
if we implement it the one or the other way we will follow the `../resources/language/resource.language.en_gb/strings.po`way 👍

**Pull request closed without merging**